content/blog/2024/04/kubewarden-1-12-release.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Kubewarden 1.12 release
+authors:
+  - Víctor Cuadrado
+date: 2024-04-24
+---
+
+Today we're glad to announce the release of Kubewarden 1.12.
+
+This release focuses on optimizations and high availability, both oriented to
+production.
+
+## Optimizing Gatekeeper policies
+
+The previous [1.11
+release](https://www.kubewarden.io/blog/2024/03/kubewarden-1-11-release/)
+featured lots of optimizations for
+[context aware policies](https://docs.kubewarden.io/next/reference/spec/context-aware-policies).
+
+The 1.12 release provides a further optimization for Gatekeeper policies that
+access Kubernetes resources. This optimization provides an extra 55%
+performance boost for these policies.
+
+This is particularly noticeable on really big clusters (thousands of nodes), as
+Gatekeeper policies expect a serialized inventory of all Kubernetes objects
+they could access, under `data.inventory`.
+
+In addition, Gatekeeper policies in Kubewarden have `spec.contextAwareResources`
+which allows us to fine-tune the context-aware permissions per policy.
+Leveraging this feature allows us to provide greater cache granularity for each
+resource set shared between policies (e.g: "all Namespaces with a specific
+LabelSelector").
```

Comment:
```suggestion
```

I would drop this section. Gatekeeper doesn't use label selectors when accessing resources, hence we cannot filter that.

What we have always been doing is to expose to the Gatekeeper policy only the resources it's interested about.

With Gatekeeper the admin determines what has to be shared with policies, then this is provided to **all** policies, even if they do not care.